ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -91,6 +91,7 @@ aaa_accounting:
     console:
       type: start-stop
       group: TACACS
+      logging: true
```

Comment:
Let's put logging on console in one of the files and on "default" in the other file. Then we test all variants. Thanks.